gorm-tools-rest/src/main/groovy/gorm/tools/rest/controller/RestRepositoryApi.groovy

```diff
@@ -230,36 +236,24 @@ trait RestRepositoryApi<D> implements RestApiController {
     List getqSearchIncludes() { [] }
 
     void handleException(RuntimeException e) {
-        //log.error e.message
         if( e instanceof EntityNotFoundException){
-            callRender(status: NOT_FOUND, e.message)
+            respond([view: '/errors/_errors'], new ApiError(status:NOT_FOUND, title: "Not Found", detail:e.message))
         }
         else if( e instanceof EntityValidationException ){
-            String defaultMessage = e.messageMap.defaultMessage as String
-            // log.info m
-            respond([view: '/errors/_errors'], [errors: e.errors, message: defaultMessage, renderArgs: [:]])
-            //callRender(status: UNPROCESSABLE_ENTITY, m)
+            respond([view: '/errors/_errors422'], new ApiValidationError(UNPROCESSABLE_ENTITY, "Validation Error", e.defaultMessage, e.errors))
         }
         else if( e instanceof ValidationException ){
-            String defaultMessage = e.message
-            respond([view: '/errors/_errors'], [errors: e.errors, message: defaultMessage, renderArgs: [:]])
+            respond([view: '/errors/_errors422'], new ApiValidationError(UNPROCESSABLE_ENTITY, "Validation Error", e.message, e.errors))
```

Comment:
what's the difference between e.defaultMessage and e.message ? Why do we use one at one point and other elsewhere ?

For ApiValidationError can title default to "Validation Error" and status to UNPROCESSABLE_ENTITY  so we don't have to copy/paste?

Reply:
@jdabal the message is full error message like ``` Field error in object 'yakworks.testify.model.Project' on field 'name': rejected value [null]; codes [yakworks.testify.model.Project.name.nullable.error```

Where as defaultMessage is `OrgSource validation errors`

Reply:
@jdabal a change done, and other question answered above